### ISSUE TITLE
Veteran Details

### ISSUE THREAD

**Issue by pacerwow** [department-of-veterans-affairs/va.gov-team#94609]
## Issue Description
Conduct an accessibility audit of the following pages:

contact-information
housing-situation
terminally-ill
alternate-names
review-veteran-details/military-service-history
review-veteran-details/military-service-history/reserves-national-guard
review-veteran-details/military-service-history/federal-orders
review-veteran-details/separation-location
separation-pay
retirement-pay
training-pay

---
## Tasks
- [ ] Conduct the accessibility audit as outlined in the Epic for the pages listed above
- [ ] Record bugs & violation in Google Sheets

## Acceptance Criteria
- [ ] An accessibility audit is performed for the pages listed above
- [ ] Results are documented in Google Sheets
- [ ] A summary is shared with our stakeholders during Sprint Review

**Comment by pacerwow** [comment]
Possible pages:

(Carbs)
supporting evidence
evidence upload

(Conditions) 
new-disabilities/add
new-disabilities/follow-up
^ Changes to both are major
